data/112/592/443/9/1125924439.geojson

```diff
@@ -84,7 +88,7 @@
         }
     ],
     "wof:id":1125924439,
-    "wof:lastmodified":1566638479,
+    "wof:lastmodified":1659047515,
     "wof:name":"\u0162urayf",
```

Comment:
Nit: lots of non-ASCII values in wof:name in this PR

Reply:
fixed in [e6a2f2d](https://github.com/whosonfirst-data/whosonfirst-data-admin-sa/pull/19/commits/e6a2f2d9a689c22d23c6ceabb5039d78bcfad61a)